backend/BjjAppApi/Services/DemosService.cs

```diff
@@ -0,0 +1,38 @@
+using BjjAppApi.Models;
+using Microsoft.Extensions.Options;
+using MongoDB.Driver;
+
+namespace BjjAppApi.Services;
+
+public class DemosService
+{
+    private readonly IMongoCollection<Demo> _demosCollection;
+
+    public DemosService(
+        IOptions<BjjAppDatabaseSettings> bjjAppDatabaseSettings)
+    {
+        var mongoClient = new MongoClient(
+            bjjAppDatabaseSettings.Value.ConnectionString);
+
+        var mongoDatabase = mongoClient.GetDatabase(
+            bjjAppDatabaseSettings.Value.DatabaseName);
```

Comment:
Can this be passed in through the constructor of the class instead of having this class instanitate these? Same for the other service classes.

Reply:
I believe it might be possible but will require further research. I will look into it once I have a working demo.